todo/src/terminal/error.rs

```diff
@@ -1,23 +1,22 @@
-use crate::repository::file_storage::error::StorageError;
+use crate::repository::todo::error::StorageError;
 use std::{fmt, io::Error};
+use uuid::Error as UuidError;
 
 #[derive(Debug)]
 pub enum TerminalError {
     Stdout(Error),
     Stdin(Error),
-    ParseInt(String),
-    IndexError,
     StorageError(StorageError),
+    UuidParse(UuidError),
 }
 
 impl fmt::Display for TerminalError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
             TerminalError::Stdin(error) => write!(f, "Input error: {}", error),
             TerminalError::Stdout(error) => write!(f, "Output error: {}", error),
-            TerminalError::ParseInt(i) => write!(f, "Parse error: \"{}\" is an invalid index!", i),
-            TerminalError::IndexError => write!(f, "Index error: Index out of bounds!"),
             TerminalError::StorageError(error) => write!(f, "Error in storage: {}", error),
+            TerminalError::UuidParse(error) => write!(f, "Error when parsing the uuid: {}", error),
```

Comment:
Eu trocaria todas as linhas daqui pra `{error}`, mas é detalhe e preferência
```suggestion
            TerminalError::UuidParse(error) => write!(f, "Error when parsing the uuid: {error}"),
```